docs/openj9_newuser.md

```diff
@@ -31,6 +31,12 @@ The Eclipse OpenJ9 virtual machine (VM) implements the [Java Virtual Machine Spe
 
 Although OpenJ9 implements its own command-line interface, many HotSpot options are recognized and accepted by the VM for compatibility. Any `-XX:` options that are not recognized by the VM are ignored by default, which prevents an application failing to start. You can turn off this behavior with the [-XX:-IgnoreUnrecognizedXXColonOptions](xxignoreunrecognizedxxcolonoptions.md) option.
 
+Set the following options for compatibility with Hotspot.
```

Comment:
After some discussion, the ID team decided that this this new table should preferably live in the "Switching to OpenJ9" (`cmdline_migration.md`) topic, at the end in a new section, something like this:
**Creating compatible behavior**
You can set the following options to make OpenJ9 behave in the same way as Hotspot.

Looking at that topic, I wondered if the `-XX:+UseNUMA` option currently in the "Equivalent options" table might arguably belong in this new table - isn't it an option you have to set to get the same default behavior as Hotspot?

Reply:
I've moved the table and added -Xnuma:none to the list.

Reply:
Thanks - all looks good